knowledge-base/grid-add-column-from-code.md

```diff
@@ -27,13 +27,13 @@ I want to add a custom Grid column programmatically. I would also like to define
 
 ## Solution
 
-You can use the [`RenderTreeBuilder`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.rendering.rendertreebuilder?view=aspnetcore-6.0) class to create a GridColumn from the C# portion of the application. 
+You can use the [`RenderFragment Delegate`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.renderfragment?view=aspnetcore-8.0) to create a GridColumn from the C# portion of the application. 
```

Comment:
Use either "Grid column" or "`GridColumn`" (styled as code because it is a class).
```suggestion
You can use the [`RenderFragment Delegate`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.renderfragment?view=aspnetcore-8.0) to create a Grid column from the C# portion of the application. 
```